app/views/home/index.html.erb

```diff
@@ -1,5 +1,43 @@
-<div class="jumbotron">
-  <h1 class="display-4">Welcome to Groupon</h1>
-  <hr class="my-4">
-  <p class="lead">Your destination for the best coupons in your city</p>
+<div class="jumbotron deals-container">
+  <h2 align="center">Deals</h2>
+  <%= form_tag(home_page_path, method: 'get', id: 'search-form', class: 'form-inline search-form') do %>
+    <%= text_field_tag :search, params[:search], placeholder: "Search Deals By Title or City", class: 'form-control search-input' %>
+    <%= collection_select(:category, :category_id, Category.all, :id, :name, {selected: params[:category]&.[](:category_id), include_blank: 'Select Category' },  data: { type: 'select2-enabled' }) %>
+    <%= submit_tag "Search", class: "btn btn-primary" %>
+  <% end %>
+  <% @deals.each do |deal| %>
+    <% if deal.published? %>
```

Comment:
why do I have to add this condition here? I should fetch only published deals in @deals